script_angular.js

```diff
@@ -409,11 +466,52 @@ app.controller('bethistory', function (myFactory) {
                                 response[i].side = response[i].away_team;
                             }
                         }
+                        if(response[i].side===response[i].away_team){
+                            response[i].line=response[i].line*-1;
+                        }
+
+                        if(response[i].bet_status==='Lost'){
+                            response[i].win_amount=response[i].win_amount*-1;
```

Comment:
should you ever be multiplying win amount by a negative number? If bet was won it will be positive. If bet was lost win amount is 0. So I don't see the purpose of this.